Sage/carbon
# fix(multi-select): fix onchange callback being called twice on option select - FE-5332

### Proposed behaviour


Fixes #5418

### Checklist




#### QA


### Additional context



### Testing instructions
https://codesandbox.io/s/carbon-quickstart-forked-zxih4h
In the provided sandbox select a new value in the MultiSelect component and check console to see how many times the `onChange handler fired` text is logged
